### ISSUE TITLE
FreeText content is not accessible

### ISSUE THREAD

**Issue by calixteman** [mozilla/pdf.js#15194]
STR:
 - create a text annotation
 - save and open the saved pdf.

The text is drawn on the canvas but there is no way for a screen reader to get it: it's a serious a11y issue.

**Comment by revolunet** [comment]
Hi, looks like it works. if i open a blank PDF, add text from the "Text" button

VoiceOver can read the text. example created 

[blank-3.pdf](https://github.com/user-attachments/files/18769012/blank-3.pdf)

file attached
